libs/ett/core/shell/src/lib/ett-shell.component.ts

```diff
@@ -35,27 +47,129 @@ const selector = 'ett-shell';
     `
       ${selector} {
         display: block;
+
+        .toolbar {
+          width: 100%;
+          display: flex;
+
+          h1 {
+            flex: auto;
+          }
+
+          .menu {
+            align-self: flex-end;
+          }
+        }
```

Comment:
issue: our components don't use emulated view encapsulation to save bandwidth, so we should use BEM-style selectors to keep styles from bleeding through to other components, for example `${selector}__toolbar`, `${selector} h1`, and `${selector}__menu`.